### DISCUSSION TITLE
I'm stuck on the proof of academic status page.

### DISCUSSION THREAD

**Discussion by ImmuneLion318** [community/community#127478] [GitHub Education] [answered]
### Hi everyone,

As I stated I'm stuck on the proof of academic status I upload an image click process it refreshes to the same page and does nothing more.

**Comment by rohispace90** [comment]
Hello dear;
Here are a few troubleshooting steps you can try:

1- Check File Format and Size.
2- Clear Browser Cache.
3- Try a Different Browser.
4- Disable Browser Extensions.
5- Check Internet Connection.
6- Rename the File.
7- Check for Website Maintenance.

Be lucky;

**Comment by davevad93** [comment, marked as answer]
Hi @ImmuneLion318 ,

Many users are experiencing similar issues, take a look at these discussions #126848 #126276 , they might be helpful.

**Comment by ImmuneLion318** [comment]
Okay thank you so much hopefully it'll be back up soon or whatever it is fixed.
